k2/csrc/array.h

```diff
@@ -410,10 +424,34 @@ class Array2 {
     that
      it will have a different memory layout than the input.
   */
-  Array2<T> To(ContextPtr ctx) {
-    // TODO
-    Array2<T> array;
-    return array;
+  Array2<T> To(ContextPtr ctx) const {
+    if (dim0_ * dim1_ == 0) {
```

Comment:
seems we can remove this if now, below code can handle the case.

Reply:
Removed.